go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -343,12 +343,12 @@ func (oc *Controller) addEgressFirewallRules(ef *egressFirewall, hashedAddressSe
 func (oc *Controller) createEgressFirewallRules(priority int, match, action, externalID string) error {
 	logicalSwitches := []string{}
 	if config.Gateway.Mode == config.GatewayModeLocal {
-		nodes, err := oc.watchFactory.GetNodes()
+		nodeLocalSwitches, err := libovsdbops.FindAllNodeLocalSwitches(oc.nbClient)
```

Comment:
@tssurya @JacobTanenbaum do we have the path covered where a node is added later? Do we look for new nodes and then add the acl? I dont see it in the node watcher.

Reply:
OK I filed https://github.com/ovn-org/ovn-kubernetes/issues/2751 for someone to fix

Reply:
My PR here https://github.com/ovn-org/ovn-kubernetes/pull/2506/files adds a node watcher for egress firewall. We could modify that with whatever might be needed here